src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -4208,10 +4208,10 @@
     },
     "System.Net.Quic": {
       "InboxOn": {
+        "netcoreapp3.0": "4.2.1.0",
```

Comment:
Why was this added? I think `System.Net.Quic` is new and will be inbox starting in `net6.0`

Reply:
i use the packageindex from release/5.0 branch as base and it seems like we have system.net.quic there in packageindex but not in repo https://github.com/dotnet/runtime/blob/release/5.0/src/libraries/pkg/baseline/packageIndex.json#L4013

i will remove this.